### PR TITLE
Update tranlation of comment in basic-javascript

### PR DESCRIPTION
**Pull request by S1ngS1ng**
## In this PR:
- Update translation of comments

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/basic-javascript.json [hunk 1]
```diff
@@ -2161,7 +2161,7 @@
         "  return \"No, it's false\";",
         "}",
         "",
-        "// Setup",
+        "// 定义 myFunction",
```
Comment (wudifeixue) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json:2164]: 哇你给的提示好多

File: seed/challenges/01-front-end-development-certification/basic-javascript.json [hunk 2]
```diff
@@ -3524,20 +3524,20 @@
...
         "  gift: \"pony\",",
         "  pet: \"kitten\",",
         "  bed: \"sleigh\"",
         "};",
         "",
         "function checkObj(checkProp) {",
-        "  // Your Code Here",
+        "  // 请把你的代码写在这条注释以下",
         "  ",
         "  return \"Change Me!\";",
```
Comment (wudifeixue) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json]: 有好几处 Change Me! 这个我觉得很有必要也翻译一下。今晚QQ群又遇见[Counting Card](https://github.com/FreeCodeCampChina/freecodecamp.cn/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20counting)老问题觉得翻译了return Change Me肯定可以有所帮助。

**Review by wudifeixue — CHANGES_REQUESTED** [review]
看看如果还有哪里有 Change Me 也一起改了吧，多谢~

File: seed/challenges/01-front-end-development-certification/basic-javascript.json (hunk 2 above)
Comment (S1ngS1ng) [reply seed/challenges/01-front-end-development-certification/basic-javascript.json]: @wudifeixue Done!

File: seed/challenges/01-front-end-development-certification/basic-javascript.json (hunk 1 above)
Comment (S1ngS1ng) [reply seed/challenges/01-front-end-development-certification/basic-javascript.json:2164]: @wudifeixue 其实这些就是翻译啦。。没啥特别的= =

**Review by wudifeixue — APPROVED** [review]

**Comment by wudifeixue** [comment]
okay

**Pull request merged**